src/logger.js

```diff
@@ -1,106 +1,106 @@
-import { is } from './utils';
-import * as errors from './errors';
-import { format as fmt } from 'util';
-import chalkModule from 'chalk';
+import { is } from './utils'
+import * as errors from './errors'
+import { format as fmt } from 'util'
+import chalkModule from 'chalk'
 
 const chalk = new chalkModule.constructor({
   enabled: process.stderr && process.stderr.isTTY,
-});
+})
 
 // Special chars.
-const chevron = '\xBB';
-const checkmark = '\u2713';
-const green = chalk.green(chevron);
-const yellow = chalk.yellow(chevron);
-const red = chalk.red(chevron);
+const chevron = '\xBB'
+const checkmark = '\u2713'
+const green = chalk.green(chevron)
+const yellow = chalk.yellow(chevron)
+const red = chalk.red(chevron)
 
 export default class Logger {
-  constructor(verbose = false, debug = false) {
-    this.verbose = verbose;
-    this._stderr = process.stderr;
-    this._debug = debug;
-    this._times = [];
+  constructor (verbose = false, debug = false) {
+    this.verbose = verbose
+    this._stderr = process.stderr
+    this._debug = debug
+    this._times = []
   }
 
   /**
    * Log arguments into stderr if the verbose mode is enabled.
    */
-  log(...args) {
+  log (...args) {
     if (this.verbose) {
-      let str = fmt(`${green} ${args.shift()}`, ...args);
-      this._stderr.write(`${str}\n`);
+      let str = fmt(`${green} ${args.shift()}`, ...args)
+      this._stderr.write(`${str}\n`)
     }
   }
 
   /**
    * Always log arguments as warning into stderr.
    */
-  warn(...args) {
-    let str = fmt(`${yellow} [WARNING] ${args.shift()}`, ...args);
-    this._stderr.write(`${str}\n`);
+  warn (...args) {
```

Comment:
Space even for class methods? I thought it was required only for annonymous functions.